web/spec/supabase.yml

```diff
@@ -675,6 +675,17 @@ pages:
             .from('messages')
             .upsert({ id: 3, message: 'foo', username: 'supabot' })
           ```
+      - name: Bulk Upsert your data
+        isSpotlight: true
```

Comment:
I think this one is better not as a spotlight (eg, make this `false`)

Reply:
@kiwicopple disabled spotlight.

Reply:
thanks!